skorch/callbacks/logging.py

```diff
@@ -875,3 +876,103 @@ def on_epoch_end(self, net, **kwargs):
 
         for key in filter_log_keys(epoch_logs.keys(), self.keys_ignored_):
             self.experiment.log_scalar(key + self.epoch_suffix_, epoch_logs[key], epoch)
+
+
+class MlflowLogger(Callback):
+    """
+    >>> import mlflow
+    >>> net = NeuralNetClassifier(net, callbacks=[MLflowLogger])
+    >>> with mlflow.start_run():
+    ...     net.fit(X, y)
```

Comment:
A few things should be addressed in the docstring:

1. add a bit of description for those users who are not familiar with MLFlow.
2. give an install instruction, `$ pip install mlflow`, since it's not part of the skorch dependencies
3. use numpydoc style.
4. document all parameters
5. since there are 2 ways of initializing, it would be nice to show examples of both and explain why you would use one or the other

For most of these points, just look at the other loggers and copy the style used there.